torch/quantization/fx/quantization_patterns.py

```diff
@@ -389,7 +390,10 @@ def convert(self, quantizer, node, load_arg, debug=False, convert_custom_config_
         emb = quantizer.modules[emb_node.target]
         qconfig = quantizer.qconfig_map[node.name]
         assert not activation_is_statically_quantized(qconfig)
-        qemb = nnq.Embedding
+        if type(emb) == torch.nn.Embedding:
```

Comment:
you can use: get_static_quant_module_class for now.. we can create a new map in the future.

Reply:
also please wait a bit until https://github.com/pytorch/pytorch/pull/46338 lands and add similar support here as well